team-validator.js

```diff
@@ -287,16 +263,16 @@ Validator = (function () {
 			return ["This is not a Pokemon."];
 		}
 
-		var template = tools.getTemplate(string(set.species));
+		var template = tools.getTemplate(Tools.getString(set.species));
```

Comment:
At this point, `getString` is no longer necessary in the validator, I believe, since packed teams aren't raw JSON.

Well, first we need to drop support for non-packed teams, if we haven't already.